core/src/test/java/cucumber/runtime/io/URLOutputStreamTest.java

```diff
@@ -4,40 +4,36 @@
 import gherkin.util.FixJava;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
-import org.webbitserver.HttpControl;
-import org.webbitserver.HttpHandler;
-import org.webbitserver.HttpRequest;
-import org.webbitserver.HttpResponse;
-import org.webbitserver.WebServer;
+import org.junit.rules.TemporaryFolder;
+import org.webbitserver.*;
```

Comment:
No star imports please, and please undo reordering of imports. Try to reduce the PR to significant changes only.